python_ta/__init__.py

```diff
@@ -22,39 +22,38 @@
 
 import pylint.lint
 import pylint.utils
+from pylint.config import VALIDATORS, _call_validator
 
 from astroid import MANAGER, modutils
 
-from .reporters import ColorReporter
+from .reporters import *
 from .patches import patch_all
 
 # Local version of website; will be updated later.
 HELP_URL = 'http://www.cs.toronto.edu/~david/pyta/'
 
+REPORTERS = (ColorReporter, PlainReporter, HTMLReporter, StatReporter)
```

Comment:
I recommend having the `reporter/__init__.py` export a tuple of the different reporter classes, rather than putting it here. (I believe `pylint` does something similar with `ALL_NODE_CLASSES`)